docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -257,12 +278,87 @@ remove_action( 'bigcommerce/template/product/archive', bigcommerce()->templates-
 add_action( 'bigcommerce/template/product/archive', 'your_callback_function', 10, 2 );
 ```
 
+## Localizing strings displayed on the storefront
+
+The language used by BC4WP to display messages on your storefront can be localized. The array `$js_i18n_array` defined in `src/BigCommerce/Assets/Theme/JS_Localization.php:19` contains all strings used by `assets/js/dist/scripts.js` to display messages on the front-end.

```

Comment:
You can localize the language used by BC4WP to display messages on your storefront. The array `$js_i18n_array` defined in `src/BigCommerce/Assets/Theme/JS_Localization.php:19` contains all strings used by `assets/js/dist/scripts.js` to display messages on the front-end.